support-frontend/assets/helpers/contributions.js

```diff
@@ -390,7 +408,7 @@ function getCustomAmountA11yHint(
   }
 
   return `Enter an amount of ${config[countryGroupId][contributionType].minInWords}
-    ${spokenCurrency} or more for your 
+    ${spokenCurrency} or more for your
```

Comment:
This function isn't actually used. There are quite a few in this file that aren't. I'll create a follow up PR to remove them all.